parallel-install/pkg/deployment/deployment.go

```diff
@@ -44,6 +45,9 @@ func NewDeployment(cfg *config.Config, ob *overrides.Builder, processUpdates fun
 
 	core := newCore(cfg, ob, kubeClient, processUpdates)
 
+	jobmanager.SetConfig(cfg)
+	jobmanager.SetKubeClient(kubeClient)
```

Comment:
Nitpick but may be a single call is enough it would be similar to rest defined here
```
	registerOverridesInterceptors(ob, kubeClient, cfg.Log)
```